apps/wallet/src/ui/app/staking/stake/StakingCard.tsx

```diff
@@ -83,16 +83,19 @@ function StakingCard() {
     const suiEarned = stakeData?.estimatedReward || 0;
 
     const [coinDecimals] = useCoinDecimals(coinType);
+    // set minimum stake amount to 1 SUI
+    const minimumStake = 1000000000;
```

Comment:
are there any utilities for creating 1 SUI in terms of a `number`? I can see this easy to get wrong haha